src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -113,6 +113,16 @@ public function denormalize($data, $type, $format = null, array $context = [])
             throw new NotNormalizableValueException(sprintf('Parsing datetime string "%s" using format "%s" resulted in %d errors: ', $data, $dateTimeFormat, $dateTimeErrors['error_count'])."\n".implode("\n", $this->formatDateTimeErrors($dateTimeErrors['errors'])));
         }
 
+        $defaultDateTimeFormat = $this->defaultContext[self::FORMAT_KEY] ?? null;
```

Comment:
Should we trigger a deprecation in case the fallback path is triggered, so ce can remove it in Symfony 7+ to enforce the `defaultContext[FORMAT_KEY]` is honored?

Reply:
Symfony 7 is released. Is it planned to remove it?

Reply:
Not yet as far as I can see. The first step to remove something in the next major release would be to deprecate it in a minor release before.